agent/handlers/v1/response.go

```diff
@@ -64,10 +66,12 @@ type VolumeResponse struct {
 // PortResponse defines the schema for portmapping response JSON
 // object.
 type PortResponse struct {
```

Comment:
If this `PortResponse` object is local to TMDE V1 and we don't plan to include port ranges for it, maybe it's better to not update the object at all?

Reply:
although, the new range fields in `PortResponse` are added in v1 TDME file, they are not really set for any TDME versions except v4.

with this feature, we plan to populate the v4 taskResponse with the requested `containerPortRange` and the corresponding dynamically assigned `hostPortRange`. the v4 code paths use the v1 struct and populate fields based on `includeV4Metadata` boolean. its not ideal, but it is what it is :) 

I hope I answered your question

Reply:
> the v4 code paths use the v1 struct

That's exactly what I was missing. Thanks :)